notebooks/223-gpt2-text-prediction/223-gpt2-text-prediction.ipynb

```diff
@@ -10,39 +10,13 @@
    "source": [
```

Comment:
<p>Core instead of IE?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/765/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>renamed, thank you</p>